### PR TITLE
gnubg, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Attempt to run a GTK based build ended in crashes.

File: games-board/gnubg/patches/gnubg-1.07.001.patchset [hunk 1]
```diff
@@ -0,0 +1,8272 @@
...
++++ b/board3d/Makefile.in
+@@ -309,7 +309,6 @@ EPOXY_LIBS = @EPOXY_LIBS@
+ ETAGS = @ETAGS@
+ EXEEXT = @EXEEXT@
+ FGREP = @FGREP@
+-FILECMD = @FILECMD@
+ FREETYPE2_CFLAGS = @FREETYPE2_CFLAGS@
+ FREETYPE2_LIBS = @FREETYPE2_LIBS@
+ GDK_PIXBUF_CSOURCE = @GDK_PIXBUF_CSOURCE@
+diff --git a/config.guess b/config.guess
```
Comment (jmairboeck) [inline games-board/gnubg/patches/gnubg-1.07.001.patchset:160]: These files from autotools don't need to be added to the patch normally, they can be generated on the fly when building (with something like `autoconf -fi` IIRC).

Comment (Begasus) [reply games-board/gnubg/patches/gnubg-1.07.001.patchset:160]: Eeps! right there, not sure how those got added.

**Comment by Begasus** [comment]
Package builds fine on both arch's, but on the 32bit machine I get a crash when ./conftest is running on a check for AVX, haven't found a way to disable running those tests and I don't know how the buildmasters react on those crashes (if they occur).

**Pull request merged**